### PR TITLE
[IMP] Toolbar: Add Icons

### PR DESCRIPTION
**Pull request by anhe-odoo**
## Task Description
The aim of this task is to add icons to all the menu items in the toolbar and in the context menu. While lots of icons has been inspired by GSheet, each one of them has been totally remade, with some icons being totally hand-made (freeze pane, insert row/col/cell, ...)

## Related Task
Odoo task ID : [3145722](https://www.odoo.com/web#id=3145722&action=333&active_id=2328&model=project.task&view_type=form&cids=1&menu_id=4720)

## Review Checklist

- [ ] feature is organized in plugin, or UI components
- [ ] support of duplicate sheet (deep copy)
- [ ] in model/core: ranges are Range object, and can be adapted (adaptRanges)
- [ ] in model/UI: ranges are strings (to show the user)
- [ ] undo-able commands (uses this.history.update)
- [ ] multiuser-able commands (has inverse commands and transformations where needed)
- [ ] new/updated/removed commands are documented
- [ ] exportable in excel
- [ ] translations (\_lt("qmsdf %s", abc))
- [ ] unit tested
- [ ] clean commented code
- [ ] track breaking changes
- [ ] doc is rebuild (npm run doc)
- [ ] status is correct in Odoo

### CODE REVIEW

**Comment by anhe-odoo** [comment]
FYI, once these icons has been approved by François, some of them will be optimized to make it lighter. But I keep them as is for the moment as it will be easier to edit if some changes are needed

File: demo/main.js [hunk 1]
```diff
@@ -78,25 +80,23 @@ class Demo extends Component {
...
     topbarMenuRegistry.addChild("read_write", ["file"], {
       name: "Open with write access",
       sequence: 13,
       isReadonlyAllowed: true,
       execute: () => this.model.updateMode("normal"),
-    });
-    topbarMenuRegistry.addChild("read_write", ["file"], {
-      name: () => (this.state.displayHeader ? "Hide header" : "Show header"),
-      isReadonlyAllowed: true,
-      action: () => (this.state.displayHeader = !this.state.displayHeader),
```
Comment (hokolomopo) [inline demo/main.js:99]: why delete ?

File: src/actions/edit_actions.ts [hunk 2]
```diff
@@ -26,40 +26,52 @@ export const copy: ActionSpec = {
...
   name: _lt("Find and replace"),
   description: "Ctrl+H",
   isReadonlyAllowed: true,
   execute: ACTIONS.OPEN_FAR_SIDEPANEL_ACTION,
+  icon: "o-spreadsheet-Icon.FIND_AND_REPLACE",
+};
+
+export const deletes: ActionSpec = {
+  name: _lt("Delete"),
+  icon: "o-spreadsheet-Icon.DELETE",
```
Comment (hokolomopo) [inline src/actions/edit_actions.ts:74]: Only used one, can probably be defined inlinde

File: src/registries/menus/topbar_menu_registry.ts [hunk 3]
```diff
@@ -67,33 +67,37 @@ topbarMenuRegistry
     sequence: 65,
     separator: true,
   })
-  .addChild("edit_delete_cell_values", ["edit"], {
-    ...ACTION_EDIT.deleteValues,
+  .addChild("deletes", ["edit"], {
```
Comment (hokolomopo) [inline src/registries/menus/topbar_menu_registry.ts:70]:
```suggestion
  .addChild("delete", ["edit"], {
```

**Review by hokolomopo — COMMENTED** [review]
Nice icons :)

**Review by LucasLefevre — APPROVED** [review]
woop woop, I like it :)
robodoo r+

**Pull request closed without merging**